beastify/popup/choose_beast.js

```diff
@@ -1,6 +1,30 @@
 /*
 Given the name of a beast, get the URL to the corresponding image.
 */
+
+/*
+updateButtons():
+* looks in the self.beast variable to see if the page is beastified or not,
```

Comment:
This won't work, because the popup's document is reloaded each time the popup gets opened, so it'll be easy for it to get out of sync with the state of the page.